lib/common/collections/wp_users/output.rb

```diff
@@ -28,18 +28,10 @@ def output(options = {})
     end
 
     def remove_junk_from_display_names
-      display_names = []
       self.each do |u|
-        display_name = u.display_name
-        unless display_name == 'empty'
-          display_names << display_name
-        end
-      end
-      junk = get_equal_string_end(display_names)
-      unless junk.nil? or junk.empty?
-        self.each do |u|
-          u.display_name ||= ''
-          u.display_name = u.display_name.sub(/#{Regexp.escape(junk)}$/, '')
+        unless u.display_name == 'empty'
```

Comment:
I removed it because I scanned some websites with all usernames ending with the same couple of characters with wrong results as display names.
In this way I checked as many website (with different themes) as I could and I always retrieved all the right display names.
This is because of the `inexistent_user` trick: it is theme dependent, but, basically, it does the same of these deleted lines, but in a different moment (before getting users list instead of after that).

Why I preferred this way? because in the worst case there will be no deletion in the title (no common words) and all the title will be printed, letting the user to understand which is the significant part.